jobs/create_manifest/create_manifest.groovy

```diff
@@ -0,0 +1,43 @@
+node(create_manifest_node){
```

Comment:
-1 for this stage.
this stage has deleteDir(), but doesn't use a new workspace.
Cause pipeline job masterci is not be restricted on specific nodes, so if this stage runs on the same node with masterci pipeline job, then the workspace of masterci will be deleted.
subsequent stages can't find their groovy scipts and will fail.